### PR TITLE
Autoclose year dropdown on selection of valid year

### PR DESCRIPTION
**Pull request by gtmsingh**
Whenever user clicks on dropdown to see the years he can select, even after choosing a particular year the year's dropdown does not goes away. I have set ```$scope.showYearsPagination``` to ```false``` whenever any valid date is set.

### CODE REVIEW

**Comment by wouldgo** [comment]
Hi @gtmsingh,

I see only the whole indentation lost. What's the actual piece of code modified?
I'm sorry but right now imho this isn't lgtm.

Thanks for your time we're open for discussion.
Dario

File: src/js/angular-datepicker.js [hunk 1]
```diff
@@ -426,10 +423,11 @@
           $scope.year = Number(year);
           setDaysInMonth($scope.monthNumber, $scope.year);
           $scope.paginateYears(year);
+          $scope.showYearsPagination = false;
```
Comment (gtmsingh) [inline src/js/angular-datepicker.js]: Here is the only line that I added.

**Comment by gtmsingh** [comment]
It was fault in my editors plugin that the whole indentation was lost. If you want the older indentation I can revert back and send you another pull request. I have commented at the line which I added. I am very new to open source contribution. If possible, can you walk me through how should I proceed with my work so that it looks structured when I send pull requests

Thanks

**Comment by wouldgo** [comment]
Hi @gtmsingh,

I'm really sorry for not responding you. Anyway yes, if you can revert the previous indentation and re-submit your contribution we really appreciated.

Have a great day
Dario.

**Comment by gtmsingh** [comment]
Hi @wouldgo,
I have tried my best to reindent the code as it was. Please check the diff and let me know if the only indentation which is not done is required or not. I don't find the reason it shows diff at that line.

Thanks!
Gautam

**Comment by wouldgo** [comment]
Hi @gtmsingh,

thanks a lot for the effort in rolling back all the indentation and for contribution! :+1: 
As soon as possible we'll publish a new version.

Best regards,
Dario

**Pull request merged**